### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alexHlebnikov**
Update Android installation instruction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,11 +48,23 @@ Run your project (Cmd+R)
 
 ### Installation (Android)
 
-* Add Gradle configuration changes
+* In `android/setting.gradle`
 
-Run `react-native link react-native-device-info` in your project root.
```
Comment (GantMan) [inline README.md]: Not sure you're modifying the correct part here.  Using `react-native link` should work.

I thought you were going to remove the directions for `rnpm`

Comment (alexHlebnikov) [inline README.md]: ```react-native link react-native-device-info``` does for me the same thing as ```rnpm install react-native-device-info``` - installs package only on iOS.

Comment (GantMan) [inline README.md]: That's odd, `react-native link` should properly link to MainApplication.java, that's actually the PR I wrote to react native.

**Comment by alexHlebnikov** [comment]
The problem still occurs on RN 0.33.0.RC.0. `react-native link` install package only on iOS.

**Pull request closed without merging**